docs/sources/datasources/azuremonitor.md

```diff
@@ -67,20 +67,22 @@ az ad sp create-for-rbac -n "http://localhost:3000"
 
 ## Choose a Service
 
-In the query editor for a panel, after choosing your Azure Monitor data source, the first option is to choose a service. There are three options here:
+In the query editor for a panel, after choosing your Azure Monitor data source, the first option is to choose a service. There are four options here:
 
-- `Azure Monitor`
+- `Metrics`
 - `Application Insights`
-- `Azure Log Analytics`
+- `Logs`
 - `Insights Analytics`
 
-The query editor changes depending on which one you pick. Azure Monitor is the default.
+The query editor changes depending on which one you pick. Metrics is the default.
 
 Starting in Grafana 7.1, Insights Analytics replaced the former edit mode from within Application Insights.
 
-## Query the Azure Monitor service
+Starting in Grafana 7.4, the Azure Monitor query type was renamed to Metrics and Azure Logs Analytics was renamed to Logs.
```

Comment:
@oddlittlebird Line 81 is an example of something that I would expect to be in release notes.

Reply:
Yes, I agree. @joshhunt, can you help us write up a short blurb for this? It does not need to be long.

Reply:
@oddlittlebird Sure! What about:

> Azure Monitor query type was renamed to Metrics and Azure Logs Analytics was renamed to Logs to match the service names in Azure and align the concepts with the rest of Grafana.

I'll let you wordsmith if need be 😅

Reply:
Perfect, thank you! I have added it to the What's New in 7.4? Gdoc.